test/integration.test.js

```diff
@@ -38,110 +37,110 @@ devConnectDescribe('dev connect', () => {
     let options;
     beforeEach(async () => {
         const keyStore = new InMemoryKeyStore(networkId);
-        const storage = createFakeStorage();
+        const storage = test_utils.createFakeStorage();
         deps = {
             keyStore,
             storage,
-            createAccount: dev.createAccountWithLocalNodeConnection
+            createAccount: test_utils.createAccountWithLocalNodeConnection
         };
         options = {
             deps,
         };
     });
 
     test('test dev connect like template', async () => {
-        window.localStorage = createFakeStorage();
+        window.localStorage = test_utils.createFakeStorage();
         // Mocking some
-        let tmpCreate = dev.createAccountWithContractHelper;
-        let devConfig = dev.getConfig;
-        dev.getConfig = async () => 'THE_CONFIG';
-        dev.createAccountWithContractHelper = async (nearConfig, newAccountId, publicKey) => {
+        let tmpCreate = test_utils.createAccountWithContractHelper;
+        let devConfig = test_utils.getConfig;
+        test_utils.getConfig = async () => 'THE_CONFIG';
```

Comment:
I think we actually don't need `getConfig` anymore, replaced by:
https://github.com/nearprotocol/near-shell/blob/master/blank_project/src/config.js